spec/requests/atmosphere/api/appliance_sets_spec.rb

```diff
@@ -132,6 +132,29 @@
         expect(response.status).to eq 403
       end
 
+      context 'when invalid appliance set type is provided' do
+        let(:invalid_type) { 'invalid appliance set type' }
+        let(:invalid_type_as) do
+          { appliance_set: {
+              name: 'my name',
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.